tools/codeGen/util/util.go

```diff
@@ -22,11 +23,15 @@ type _UnusedCompileAssert2 = [8 - unsafe.Sizeof(int(0))]byte
 type UVarint = uint64
 type Varint = int64
 type UInt = uint64
+type UInts = []uint64 // Subject to change when array support comes.
```

Comment:
there is array support, did you try: `[UInt]` ?

Reply:
Nope. I missed the memo.